### PR TITLE
Updated deprecated attribute

### PR DESCRIPTION
**Pull request by richarth**
The singleLine attribute is deprecated so I've swapped that with maxLines=1 instead.

### CODE REVIEW

**Comment by richarth** [comment]
I signed it!

File: app/src/main/res/layout/fragment_addnote.xml [hunk 1]
```diff
@@ -32,7 +32,7 @@
                 android:layout_width="match_parent"
                 android:layout_height="wrap_content"
                 android:hint="@string/title_hint"
-                android:singleLine="true"
+                 android:maxLines="1"
```
Comment (slinzner) [inline app/src/main/res/layout/fragment_addnote.xml]: nit: rm whitespace

**Comment by richarth** [comment]
I've removed the extra space

**Comment by slinzner** [comment]
Thanks!

**Pull request merged**